### ISSUE TITLE
TV watch_providers does not return response

### ISSUE THREAD

**Issue by georgep9** [celiao/tmdbsimple#97]
The TV module function `watch_providers` does not return a response value.

Here's a snippet of the function in [`/tmdbsimple/tv.py`](https://github.com/celiao/tmdbsimple/blob/master/tmdbsimple/tv.py)
```python
...
    def watch_providers(self, **kwargs):
        """
        Get a list of the availabilities per country by provider for tv.

        Args:
            None

        Returns:
            A dict respresentation of the JSON returned from the API.
        """
        path = self._get_id_path('watch_providers')

        response = self._GET(path, kwargs)
        self._set_attrs_to_values(response)
...
```

**Comment by georgep9** [comment]
Fixed in #98

**Comment by georgep9** [comment]
duplicate of #88 and PR #98

**Comment by celiao** [comment]
✅ Fixed in [commit 46c9422](https://github.com/celiao/tmdbsimple/commit/46c9422) — `return response` added to `TV.watch_providers()`, along with an updated test to prevent regressions.
